tests/pytest.ini

```
@@ -19,12 +19,5 @@ markers =
     logging: marks tests for pennylane logging
     external: marks tests that require external packages such as matplotlib and PyZX
     catalyst: marks tests for catalyst testing (select with '-m "catalyst"')
-filterwarnings =
-    ignore::DeprecationWarning:autograd.numpy.numpy_wrapper
-    ignore:Casting complex values to real::autograd.numpy.numpy_wrapper
-    ignore:Casting complex values to real discards the imaginary part:UserWarning:torch.autograd
-    ignore:Call to deprecated create function:DeprecationWarning
-    ignore:the imp module is deprecated:DeprecationWarning
-    error::pennylane.PennyLaneDeprecationWarning
-addopts = --benchmark-disable
+addopts = -W error --benchmark-disable
```

```suggestion
addopts = --benchmark-disable
```

Favour use of env-var instead